nix/overlay.nix

```diff
@@ -55,6 +55,11 @@ in {
     configureFlags = old.configureFlags ++ [ "--enable-prof" "--enable-stats" ];
   });
   broker = final.callPackage ./broker {inherit stdenv; python = final.python3;};
+  simdjson = prev.simdjson.overrideAttrs (old: {
```

Comment:
There are plans for static-lib as the default option for build, but currently simdjson uses shared-lib. So a little adjustment is required.

https://github.com/simdjson/simdjson/issues/1199